src/@types/expressMiddleware.ts

```diff
@@ -0,0 +1,31 @@
+import { RedisOptions } from 'ioredis';
```

Comment:
This file should be renamed `expressMiddleware.d.ts`

Reply:
I have done this. I won't push changes until a few PR's close because I've already taken the time to pull them and work them and make them compatible.